lifelines/fitters/weibull_aft_fitter.py

```diff
@@ -169,3 +169,54 @@ def predict_cumulative_hazard(self, X, times=None, ancillary_X=None):
         lambda_, rho_ = self._prep_inputs_for_prediction_and_return_scores(X, ancillary_X)
 
         return pd.DataFrame(np.outer(times, 1 / lambda_) ** rho_, columns=_get_index(X), index=times)
+
+    def predict_cumulative_hazard_at_single_time(self, X, times, ancillary_X=None):
```

Comment:
I'd like to not include these in this PR (I assume it wasn't intended to be included?)

Reply:
Whoops, yeah I was going to submit that as my next PR (after our conversation on SO), would you want a PR for that, or do you want to implement that functionality differently.

Reply:
I would like to think further about how to include this functionality, so let's leave it out for now.